src/config.ts

```diff
@@ -102,7 +102,7 @@ export function getDefaultConfig<State>() {
                     },
                     {
                         redirect: true,
-                        track: config.nostoAnalytics ? "serp" : undefined,
+                        track: (config.nostoAnalytics ? "serp" : undefined)!,
```

Comment:
This signature looks a bit weird. The condition returns undefined but we are using bang operator on top of it